internal/alloycli/automemlimit_linux.go

```diff
@@ -0,0 +1,12 @@
+package alloycli
```

Comment:
Shouldn't we have a `//go:build linux`? Otherwise, on a non-Linux, wouldn't Go try to build both files?

Reply:
The `linux` suffix behaves like go build linux.